service/k8sclient/k8sclient.go

```diff
@@ -326,12 +325,35 @@ var (
 	ErrNotFound error = errors.New("resource was not found in Kubernetes cluster")
 )
 
+var pmmClientImage string
+
 // K8sClient is a client for Kubernetes.
 type K8sClient struct {
 	kubeCtl *kubectl.KubeCtl
 	l       logger.Logger
 }
 
+func init() {
```

Comment:
Would it be better to move this to `k8sclients.New`?
Advantage is we need to decide only once when placed in init. It does not matter I think because version does not change on runtime.